links/plan.sh

```diff
@@ -0,0 +1,264 @@
+# This file is the heart of your application's habitat.
+# See full docs at https://www.habitat.sh/docs/reference/plan-syntax/
+
+# Required.
+# Sets the name of the package. This will be used in along with `pkg_origin`,
+# and `pkg_version` to define the fully-qualified package name, which determines
+# where the package is installed to on disk, how it is referred to in package
+# metadata, and so on.
+pkg_name=links
+
+# Required unless overridden by the `HAB_ORIGIN` environment variable.
+# The origin is used to denote a particular upstream of a package.
+pkg_origin=smacfarlane
```

Comment:
Or force push a fix post-coffee!

![it's probably fine](https://user-images.githubusercontent.com/517302/27694142-4985f35c-5cb9-11e7-91b6-d6bc5940b178.gif)